templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +24,12 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <th>
+                <a href="{% url 'taxi:manufacturer-update' manufacturer.id %}">UPDATE</a>
+              </th>
+              <th>
+                <a href="{% url 'taxi:manufacturer-delete' manufacturer.id %}">DELETE</a>
+              </th>
```

Comment:
The `<th>` tags are used for table headers, but here they are used for regular table cells. Consider replacing `<th>` with `<td>` for the 'Update' and 'Delete' action links to maintain semantic HTML.